fast64_internal/oot/__init__.py

```diff
@@ -88,6 +89,7 @@ def oot_register(registerPanels):
     bpy.types.Scene.ootBlenderScale = bpy.props.FloatProperty(name="Blender To OOT Scale", default=10)
     bpy.types.Scene.ootActorBlenderScale = bpy.props.FloatProperty(name="Blender To OOT Actor Scale", default=1000)
     bpy.types.Scene.ootDecompPath = bpy.props.StringProperty(name="Decomp Folder", subtype="FILE_PATH")
+    bpy.types.Scene.ootHackerFeaturesEnabled = bpy.props.BoolProperty(name="Enable HackerOOT Features")
```

Comment:
I added hackerFeaturesEnabled to the OOT_Properties, but since ootBootupSceneOptions are settings specifically for the scene export operator, I thought it would be better to leave it as is.

Reply:
The purpose of `scene.fast64.oot` is avoiding bloating bpy.types.Scene, I think it'd be fine if it contains export stuff, it's just a different location for data stored on the same data-block in the end (the scene)

But no big deal, it's okay

Reply:
Oh, I see what you mean, its supposed to be for all OOT related properties. I thought it was intended for OOT globally relevant settings. In that case I'll add the scene bootup options in.